solarforecastarbiter/reports/templates/metrics_meta_table_text

```diff
@@ -1,16 +1,19 @@
-<p>The table below shows the normalization and deadband parameters
-used in calculating the metrics. The <em>normalization</em> factor
-is applied when calculating the metrics NMAE, NMBE, and NRMSE. The
-normalization is in the same units as the forecast and observation. By
-default, AC power forecasts are normalized by AC capacity and DC power
-forecasts are normalized by DC capacity. Normalization for all other
-forecasts is undefined, and the metric values are set to <code>nan</code>.
-The <em>deadband</em> accounts for observation uncertainty by setting
-the error (forecast - observation) equal to 0 for any point that is
-within the deadband. The error is unchanged for any point that is outside
-the deadband. The deadband is specified as a percentage of the
-observation value at each time. A value of <code>None</code>
-indicates that no deadband was applied for that observation/forecast pair.
-The deadband is accounted for in the following metrics:
-MAE, MBE, RMSE, MAPE, NMAE, NMBE, NRMSE. It is ignored for all other metrics.
-</p>
+<p>The table below shows the normalization, deadband, and cost
+parameters used in calculating the metrics. The <em>normalization</em>
+factor is applied when calculating the metrics NMAE, NMBE, and
+NRMSE. The normalization is in the same units as the forecast and
+observation. By default, AC power forecasts are normalized by AC
+capacity and DC power forecasts are normalized by DC
+capacity. Normalization for all other forecasts is undefined, and the
+metric values are set to <code>nan</code>.  The <em>deadband</em>
+accounts for observation uncertainty by setting the error (forecast -
+observation) equal to 0 for any point that is within the deadband. The
+error is unchanged for any point that is outside the deadband. The
+deadband is specified as a percentage of the observation value at each
+time. A value of <code>None</code> indicates that no deadband was
+applied for that observation/forecast pair.  The deadband is accounted
+for in the following metrics: MAE, MBE, RMSE, MAPE, NMAE, NMBE, NRMSE,
+Skill, Cost.  It is ignored for all other metrics.  See the
```

Comment:
To parallel the other items, maybe add sentence like "The <em>cost parameter</em> is the set of cost models used to calculate the Cost metric. See..."